mmcv/fileio/file_client.py

```diff
@@ -74,17 +105,143 @@ def __init__(self, path_mapping=None, enable_mc=True):
         assert isinstance(path_mapping, dict) or path_mapping is None
         self.path_mapping = path_mapping
 
-    def get(self, filepath):
-        filepath = str(filepath)
+    def _path_mapping(self, filepath: str) -> str:
```

Comment:
Maybe `_map_path`, since mapping is a noun while this is a function. This same rule is used in `_format_path`.